source/site-administrators/engine/engine-headers-based-authentication.rst

```diff
@@ -180,12 +180,12 @@ To add optional attributes, add the following inside the ``<headers>`` tag:
 
 where:
 
-* **name**: The name of the attribute in the header.  The ``APP_`` prefix shown above is just an example and could be
-  anything.
-* **field**: The name of the attribute in the authentication object.
+* **name**: The name of the attribute in the header, with the prefix removed.  (if your prefix is ``CRAFTER_`` then the 
+header value would be ``CRAFTER_first_name``, and you should enter ``first_name`` in this tag.)
+* **field**: The name of the attribute that will be created in the authentication object.
 
-To get the value of the attribute passed in the header, use the following ``authToken.principal.attributes.FIELD_NAME``,
```

Comment:
`FIELD_NAME` is supposed to be a placeholder, I'm not sure if using the value from the example is good for this general note. Maybe it would be better to just make more notorious that it is a placeholder: `<YOUR_FIELD_NAME>`?

Reply:
That would work.  I did try to tie the value in the authToken to the value in the XML above, and we can do this with <YOUR_FIELD_NAME>